src/cryptonote.php

```diff
@@ -29,7 +29,7 @@ public function sc_reduce($input)
         {
             $integer = $this->ed25519->decodeint(hex2bin($input));
             
-            $modulo = bcmod($integer , $this->ed25519->l);
+            $modulo = gmp_mod($integer , $this->ed25519->l);
```

Comment:
This could probably be left alone. Otherwise add some kind of logic to check that gmp is actually installed, otherwise this will fail on systems that only have bcmath

Reply:
Right you are, sorry